Content.Shared/Contraband/ShowContrabandSystem.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Shared.Inventory;
+
+namespace Content.Shared.Contraband;
+
+public sealed class ShowContrabandSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<ShowContrabandDetailsComponent, InventoryRelayedEvent<GetContrabandDetailsEvent>>(GetContrabandDetailsEventHandler);
```

Comment:
```suggestion
        Subs.SubscribeWithRelay<ShowContrabandDetailsComponent, GetContrabandDetailsEvent>(OnGetContrabandDetails);
```
we can subscribe to both the direct event and the relay like this
that way we don't need to directly check for `ShowContrabandDetailsComponent` in the other system